src/design/content-designer/beta.md

```diff
@@ -15,6 +15,6 @@ During the Beta phase, the product is closer to its final form, and the focus is
 - working with the team to decide whether further changes are required to fine tune the final product
```

Comment:
Changes look good, nit picking: we need to try and follow our own standards when creating branches and commits [Naming conventions](https://nhsbsa.github.io/nhsbsa-digital-playbook/development/coding-naming-conventions/)